raiden/network/transport/matrix/transport.py

```diff
@@ -1053,13 +1060,19 @@ def _get_private_room(self, invitees: List[User]) -> Room:
         self.log.debug("Creating private room", room=room, invitees=invitees)
         return room
 
-    def _get_public_room(self, room_name: str, invitees: List[User]) -> Room:
+    def _get_public_room(self, invitees: List[User]) -> Room:
         """ Obtain a public, canonically named (if possible) room and invite peers """
+        # retrieve address from invitees implicitly, is constant for all invitees
+        assert self._raiden_service is not None  # Extra assertion for mypy
+        address = validate_userid_signature(invitees[0])
+        address_pair = sorted(
+            [to_normalized_address(address) for address in [address, self._raiden_service.address]]
```

Comment:
IMO it is bad style, even though it does work in python3 it is a red flag for me. Please don't use the a variable name in the comprehension loop that is also used in the outer scope.

Reply:
Btw, why not just `sorted((to_normalized_address(address), to_normalized_address(self._raiden_service.address)))`? Having a loop here is just unnecessary.

Reply:
True